modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Install and update functions for the OE Whitelabel Paragraphs module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\oe_bootstrap_theme\ConfigImporter;
+
+/**
+ * Implements hook_install().
+ *
+ * Customise fields for whitelabel paragraphs.
+ */
+function oe_whitelabel_paragraphs_install($is_syncing): void {
+  // If we are installing from config, we bail out.
+  if ($is_syncing) {
+    return;
+  }
+
+  $configs = [
+    'core.entity_form_display.paragraph.oe_links_block.default',
```

Comment:
please sort this list, and I see 'field.field.paragraph.oe_social_media_follow.field_oe_social_media_variant.yml' is missing ?